bgpd/bgpd.h

```diff
@@ -1270,7 +1270,7 @@ struct peer {
 	 * so if a flag is unset, the corresponding override flag is unset too.
 	 * However if a flag is set, the corresponding override flag is set.
 	 */
-	uint32_t flags_override;
+	uint64_t flags_override;
```

Comment:
I know it’s not likely that we’ll need to make these bigger again, but can you please put a note here and in peer->af_flags peer->flags that these data structures are linked and should be considered in parallel?

Reply:
Added a note.